common/js/src/executable-module/executable-module.js

```diff
@@ -0,0 +1,77 @@
+/** @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview This file contains an abstract interface that can be used for
+ * loading and interacting with a binary executable module (e.g., an
+ * Emscripten/WebAssembly module).
+ */
+
+goog.provide('GoogleSmartCard.ExecutableModule');
+
+goog.require('goog.log.Logger');
+goog.require('goog.messaging.AbstractChannel');
+goog.require('goog.Disposable');
+goog.require('goog.Promise');
```

Comment:
Agreed. The Style Guide changed a lot in the last couple of years, so the whole codebase in this project needs the migration to the up-to-date practices. I've filed #264 to track this. Will not try to address this in the current PR now, since that JavaScript migration will be a separate effort.